cpp/util/etcd.cc

```diff
@@ -918,5 +966,17 @@ void EtcdClient::Generic(const string& key, const string& key_space,
                       bind(&EtcdClient::FetchDone, this, etcd_req, _1)));
 }
 
+list<EtcdClient::HostPortPair> SplitHosts(const string& hosts_string) {
+  vector<string> hosts(util::split(hosts_string, ','));
+  CHECK(!hosts.empty());
```

Comment:
Maybe print something (`<< "blabla"`) here? This could happen with invalid flags, best to be as informative as we can.